engfrosh_site/management/views.py

```diff
@@ -424,200 +419,6 @@ def manage_frosh_teams(request: HttpRequest) -> HttpResponse:
     return HttpResponseBadRequest()
 
 
-@permission_required("discord_bot_manager_discordchannel.change_discordchannel")
-def manage_discord_channels(request: HttpRequest) -> HttpResponse:
```

Comment:
Why are you removing all these management pages? Most of these pages were made to meet a need, I'm happy for them to be moved somewhere else, but their functionality does need to still exist somewhere.

Reply:
Some of their functionality is being moved (locking and unlocking will be added to the bot sometime later today) but some of these seem kind of redundant. The `manage_discord_servers` view doesn't seem very useful for creating servers as its definitely easier to manually create a server than to have the site use an OAuth token to call it for you. Other than that its functionality can be replaced with an environment variable setting guild ids (probably will also be implemented today as I need to look into why its not sending messages to discord channels). Adding admins to the server also isn't really needed in the site as the role it creates should only be for directors as it has the `administrator` permission.

Reply:
A lot of the time it's useful to not have to go into discord to lock and unlock channels, so keeping the functionality to be able to do so via the website would be good. It's especially useful on things like mobile, etc. I'm fine if you want to rewrite it, but there should still be a button at this level.

Reply:
That's fair, this has been reverted but I've left a todo not on it to make some changes as I'm not 100% sure it works and the way team channels are structured has changed a bit